rpcs3/rpcs3qt/save_data_list_dialog.cpp

```diff
@@ -94,10 +24,9 @@ save_data_list_dialog::save_data_list_dialog(const std::vector<SaveDataEntry>& e
 	//m_list->setItemDelegate(new table_item_delegate(this)); // to get rid of cell selection rectangles include "table_item_delegate.h"
 	m_list->setSelectionMode(QAbstractItemView::SelectionMode::SingleSelection);
 	m_list->setSelectionBehavior(QAbstractItemView::SelectRows);
-	m_list->setEditTriggers(QAbstractItemView::NoEditTriggers);
 	m_list->setContextMenuPolicy(Qt::CustomContextMenu);
-	m_list->setColumnCount(3);
-	m_list->setHorizontalHeaderLabels(QStringList() << tr("Title") << tr("Subtitle") << tr("Save ID"));
+	m_list->setColumnCount(4);
```

Comment:
I saw a method the other day (I believe it was citra)
make an enum with all the Column indices and add a YourEnumLength to the end, then this value will be good as long as you use all the indices for columns